### PR TITLE
Don't reload on enter find usages

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?242019 @proteusvacuum

### CODE REVIEW

File: src/core.js [hunk 1]
```diff
@@ -746,7 +746,7 @@ define([
             return false;
         });
 
-        $modalBody.find('#findUsagesSearch').on('keyup inserted.atwho', function () {
+        $modalBody.find('#findUsagesSearch').on('keyup keypress inserted.atwho', function (e) {
```
Comment (orangejenny) [inline src/core.js]: Might as well remove the `keyup` rather than firing two events for each character.

**Comment by orangejenny** [comment]
Looks fine, made a couple picky comments.

**Comment by orangejenny** [comment]
Thought about this for another minute...it'd be cleaner / more semantic to make the `form` in `find_usages_search.html` a div, since it isn't really a form, and that would fix this problem without adding js. Then would just need to tweak the `.fd-search` styles in `forms.less` to target `.form-inline` instead of `form` elements.

**Comment by emord** [comment]
Yeah. it looks nicer in code now. I originally did this as that's how the search on the case properties tree works. I attempted to change that too, but changing it changed the css somehow, so leaving that for another day

File: src/core.js (hunk 1 above)
Comment (emord) [inline src/core.js]: Gah. I was on the wrong branch and accidentally pushed `debounce` to master https://github.com/dimagi/Vellum/commit/65cccf5d3ccc676cedc05b8d89247b3e3293fb68 I'm not going to worry about reverting it right now as it's a small change, but that should also be reviewed

**Comment by orangejenny** [comment]
changes look good

**Pull request merged**